cwms_radar_api/src/main/java/cwms/radar/security/CwmsAccessManager.java

```diff
@@ -153,14 +147,15 @@ private Set<RouteRole> getRoles(Principal principal) {
         Set<RouteRole> retval = new LinkedHashSet<>();
         if (principal != null) {
             List<String> roleNames;
-            try {
+            //try {
                 CwmsUserPrincipal cup = (CwmsUserPrincipal) principal;
                 roleNames = cup.getRoles();
-            } catch (ClassCastException e) {
+            //} 
+            /*catch (ClassCastException e) {
                 // The object is created by cwms_aaa with the cwms_aaa classloader.
                 // It's a CwmsUserPrincipal but it's not our CwmsUserPrincipal.
                 roleNames = callGetRolesReflectively(principal);
-            }
+            }*/
```

Comment:
Just remove the the try/catch if you're commenting out everything in the catch branch.   Or replace the reflexive stuff with a WARNING message that the cwms_aaa jar needs to be in the tomcat lib dir or whatever the specific fix is.